arcgishub/discussions.py

```diff
@@ -166,7 +166,7 @@ def update(self, body=None, title=None, discussion=None, geometry=None, appInfo=
         if appInfo:
             payload['appInfo'] = appInfo
         
-        url = 'http://hub.arcgis.com/api/discussions/v1/posts/{}'.format(self.id)
+        url = 'http://hubqa.arcgis.com/api/discussions/v1/posts/{}'.format(self.id)
```

Comment:
is there a particular reason we are pointing this to qa instead of prod?

Reply:
@ManushiM There were 2 commits in this PR and looks like you viewed the first commit. In the second commit, you will see that I have the prod endpoints in the `Files Changed` tab. 

I have also removed the print statements from the module and updated the notebook accordingly.

Reply:
ah I just noticed. Looks good to merge!